mgradm/cmd/scale/scale.go

```diff
@@ -0,0 +1,54 @@
+// SPDX-FileCopyrightText: 2024 SUSE LLC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package scale
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/uyuni-project/uyuni-tools/shared"
+	. "github.com/uyuni-project/uyuni-tools/shared/l10n"
+	"github.com/uyuni-project/uyuni-tools/shared/types"
+	"github.com/uyuni-project/uyuni-tools/shared/utils"
+)
+
+type scaleFlags struct {
+	Backend  string
+	Replicas int
+}
+
+func addScaleFlags(cmd *cobra.Command) {
+	cmd.Flags().Int("replicas", 0, L("How many replicas of a container should be started."))
+}
+
+// NewCommand adjusts a containers replicas.
+func NewCommand(globalFlags *types.GlobalFlags) *cobra.Command {
+	scaleCmd := &cobra.Command{
+		Use:     "scale",
+		GroupID: "management",
+		Short:   L("Adjust the replicas for container that supports it."),
+		Long:    L("Adjust the replicas for container that supports it."),
```

Comment:
Shouldn't we add a list of containers that supports it in the Long help? I already see people complaining because `mgradm scale --replicas 2 uyuni-server`  doesn't work

Reply:
makes sense. how about 
`Adjust the replicas for containers that support it. Supported containers: uyuni-server-attestation` ?

Reply:
You can use multiline strings using backticks. After thinking about the wording again, we should probably name those differently than containers: in the kubernetes world this would address a `deployement` or `replicaSet` and the `container` is a different concept. Service isn't much better since it is the thing that exposes the network ports of the pods in the cluster...

```suggestion
		Short:   L("Adjust the replicas for services that supports it."),
		Long:    L(`Adjust the replicas for services that supports it.

Supported services:
  - uyuni-server-attestation
`),
```

Reply:
it will always be confusing since the terminology of one backend will never match the other backend.

Reply:
> it will always be confusing since the terminology of one backend will never match the other backend.

Yes, but you'll have multiple containers for the same coco attestation, so even for `podman` case that would be confusing as the real containers will be `coco-attestation-%d`